docs/release-notes/NuGet-5.0-RTM.md

```diff
@@ -0,0 +1,156 @@
+---
+title: NuGet 5.0 RTM Release Notes
+description: Release notes for NuGet 5.0 including known issues, bug fixes, new features, and DCRs.
+author: karann-msft
+ms.author: karann
+ms.date: 04/02/2019
+ms.topic: conceptual
+---
+
+# NuGet 5.0 Release Notes
+
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version| Available in .NET SDK(s)|
+|:---|:---|:---|
+| [**5.0.0**](https://nuget.org/downloads) | [Visual Studio 2019 version 16.0](https://visualstudio.microsoft.com/downloads/) | 2.1.602<sup>1</sup>, 2.2.202<sup>2</sup> |
+
+<sup>1</sup>Installed with Visual Studio 2019 with .NET Core workload 
+
+<sup>2</sup>Available as an optional install with Visual Studio 2019 with .NET Core workload
+
+## Summary: What's New in 5.0
+
+* Support solution filter when restoring the solution via CLI or in Visual Studio - [#5820](https://github.com/NuGet/Home/issues/5820)
+* `BuildTransitive` folder enables packages to transitively contribute targets/props to the host project - [#6091](https://github.com/NuGet/Home/issues/6091)
+* Better support for PackageReference scenarios in NuGet IVs APIs - [#7005](https://github.com/NuGet/Home/issues/7005), [#7493](https://github.com/NuGet/Home/issues/7493)
+* `dotnet pack project.json` has been deprecated - [#7928](https://github.com/NuGet/Home/issues/7928)
+* Gen 1 Credential Provider plugin is no longer supported - [#7819](https://github.com/NuGet/Home/issues/7819)
+
+## Issues fixed in this release
+
+**Bugs**
+
+* When doing a NoOp restore, avoid obj\*dgspec.json write - [#7854](https://github.com/NuGet/Home/issues/7854)
```

Comment:
This isn't properly rendered on the UI: https://review.docs.microsoft.com/en-us/nuget/release-notes/nuget-5.0-rtm?branch=pr-en-us-1378#issues-fixed-in-this-release

Maybe `obj\\*dgspec.json`?

Reply:
based on that feedback, i changed the title of the issue to: When doing a NoOp restore, avoid *.dgspec.json write in obj directory

@karann-msft  - if that sounds good to you, please manually update the readme change with that title.

Reply:
@karann-msft  - I changed the title of the bug a bit... perhaps just go use that.

Reply:
updated to use the new title.